### DISCUSSION TITLE
Discussion re version 3.3 (and v 3.2)

### DISCUSSION THREAD

**Discussion by eeholmes** [nmfs-opensci/GitHub-Guide#3] [Team Posts]
I edited the intro of the SOP because after getting Rick's feedback and Josh and my discussions last week, I realized that it sounded confrontational. The goal is to help staff understand and follow guidelines, but there are also lot of interpretation problems with the guidelines. Here is a presentation from PMEL. Not sure sure when I am guessing around 2018 or so. https://drive.google.com/file/d/1zdSesY5QrkXHD2RVW_e8Wewed1gQf-Ki/view

I started this thread to try out using GitHub discussions instead of email. This is just for the GitHub-SOP doc and mainly for those working on it.  The Visioning thread is on email.

**Comment by k-doering-NOAA** [comment]
Thanks, Eli! By Github-SOP, we are talking about [the nmfs-opensci SOP](https://nmfs-opensci.github.io/GitHub-SOP/), correct?

**Comment by eeholmes** [comment]
Yes. This discussion thread is just specific to the nmfs-opensci SOP not the bigger vision discussion or any other SOPs. 

Over the weekend, I did another update to the nmfs-opensci SOP to make it clearer that this isn't a **new** SOP. This is trying to give pragmatic guidance and makes clear that local centers/offices might have their own SOPs.  I added more links to the existing guidance too.
